### ISSUE TITLE
Notification 'sender' parameter is blank

### ISSUE THREAD

**Issue by abranson** [rogora/hangish#40]
Hi,

In our pebble app we're storing the sender parameter of the Notify call to let users filter notification, but Hangish leaves this blank. I think this also means it's missing the title on the notification popup. I had a look, but can't see where this gets defined in the nemo plugin. If possible, this should say 'Hangish'.

**Comment by rogora** [comment]
Yes, it is blank; I never actually found a real documentation for the notifications in sailfish, so i had to play with the API and see what happened.
I think the title is not missing though; anyway, the notification is created and published in notifier.cpp.
I hope I can find the time to update hangish and fix these recent bugs soon

**Comment by abranson** [comment]
Thanks, I'll keep looking myself - just thought it would be a good idea to log the issue anyway. It must be there somewhere.
